internal/websocket/common/hub_pool_test.go

```diff
@@ -54,10 +54,11 @@ func TestCreateRemoveConcurrently(t *testing.T) {
 	hubs := &sync.Map{}
 
 	wg := sync.WaitGroup{}
-	wg.Add(channelsNo * clientsPerChannel)
+	wg.Add(channelsNo*clientsPerChannel + channelsNo)
```

Comment:
Caan you explain why this logic? Preferably in comment in code.

Reply:
Explained in comment